packages/orchestration/src/utils/start-helper.js

```diff
@@ -56,7 +56,7 @@ import { makeZcfTools } from './zcf-tools.js';
  * @param {ZCF} zcf
  * @param {Baggage} baggage
  * @param {OrchestrationPowers} remotePowers
- * @param {ERemote<EMarshaller>} marshaller
+ * @param {ERemote<EMarshaller>} remoteMarshaller
```

Comment:
If it's meant to be the remote marshaller, let's reflect that in the types too (rewrapping a marshaller is fine, it's just wasteful)

```suggestion
 * @param {Remote<Marshaller>} remoteMarshaller
```

Reply:
Good idea. Thanks for removing the automerge.